### DISCUSSION TITLE
8+ GB of memory usage for a fresh searxng/searxng docker pull?

### DISCUSSION THREAD

**Discussion by arostad** [searxng/searxng#2359] [Q&A]
Is this normal? Seems like a ton of overhead for such a simple application?

*Installing in unRAID via the Community Applications SearXNG template that pulls from searxng/searxng

<img width="772" alt="Screenshot 2023-04-19 at 8 26 20 PM" src="https://user-images.githubusercontent.com/10327486/233252788-bc09648a-d30a-440f-a2a0-eb010f267785.png">

**Comment by RedSnt** [comment]
That amount of % is surprisingly similar to what my searxng instance is running at from a fresh install as well. My system is only 64GB though, and Docker limit of half that amount:
![image](https://github.com/searxng/searxng/assets/6242718/f0ba83ad-4b17-4187-92b3-4162672db7db)

Funny how it's almost 7% on the dot.

**Reply by unixfox** [reply]
docker display the memory usage with the cache and everything else. for the real memory usage check the RES memory, not through `docker stats`.
see more about it here: https://www.linuxatemyram.com/
